docs/core/proto-docs.md

```diff
@@ -5884,6 +5888,24 @@ Params represents the parameters used for by the slashing module.
 
 
 
+<a name="cosmos.slashing.v1beta1.SlashValidatorProposal"></a>
+
+### SlashValidatorProposal
+
+
```

Comment:
Could you briefly describe the edge cases here? (E.g. can also slash inactive proposers, slash happens to live stakers + unbonders at time of vote end)

Reply:
okey, make sense